tests/test_premium_accounts.py

```diff
@@ -0,0 +1,119 @@
+from unittest import mock
+from unittest.mock import patch
+
+import pytest
+
+from nordigen.api import PremiumAccountApi
+
+
+class TestPremiumAccountApi:
+    """Test Account api"""
+
+    account_id = "1d2b827b-9ca2-4adb-b4c3-0deb76a0ac50"
+    iban = "LT213250024324970797"
+
+    @pytest.fixture(scope="class")
+    def account(self, client) -> PremiumAccountApi:
+        """
+        Create Account api instance.
+
+        Args:
+            client (NordigenClient): return NordigenClient instance
+
+        Returns:
+            PremiumAccountApi: return PremiumAccountApi instance
+        """
+        return PremiumAccountApi(client=client, id=self.account_id)
+
+    def test_get_account_metadata(self, account: PremiumAccountApi):
```

Comment:
As far as I can see `account.get_metadata()` receives status 404 for premium endpoints.